handler_models.go

```diff
@@ -257,15 +259,14 @@ func ModelOwnerVersionZip(owner, name string, user *users.User, tx *gorm.DB,
 	// commit the DB transaction
 	// Note: we commit the TX here on purpose, to be able to detect DB errors
 	// before writing "data" to ResponseWriter. Once you write data (not headers)
-	// into it the status code is set to 200 (OK).
+	// into it the status code is set to 302 (Found).
 	if err := tx.Commit().Error; err != nil {
 		return nil, gz.NewErrorMessageWithBase(gz.ErrorZipNotAvailable, err)
 	}
 
-	// Serve the zip file contents
-	// Note: ServeFile should be always last line, after all headers were set.
-	http.ServeFile(w, r, *zipPath)
-	return nil, nil
+	// Redirect to the cloud storage
```

Comment:
Since the storage can change, let's keep it generic.

```suggestion
	// Redirect to the storage containing the file for download.
```